generator/ServiceClientGeneratorLib/GeneratorDriver.cs

```diff
@@ -812,7 +812,6 @@ public static void UpdateUnitTestProjects(GenerationManifest generationManifest,
         /// </summary>
         public static List<string> UpdateUnitTestProjects(GenerationManifest generationManifest, GeneratorOptions options, string serviceTestFilesRoot, ServiceConfiguration serviceConfiguration)
         {
-            Console.WriteLine("Updating unit test project files.");
```

Comment:
This method calls an overload of `UpdateUnitTestProjects` which _also_ writes the same message, so I removed this to avoid duplication.